charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -61,6 +61,10 @@ spec:
             secretKeyRef:
               name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}
               key: personalAPIKey
+        {{- if $.Values.proxy }}
```

Comment:
As we discussed offline, I would leverage common.proxy before merging

Reply:
Updated in 863e57c2bcf0c0f9f1855ce05dba33158d91b85d